src/librustdoc/clean/inline.rs

```diff
@@ -290,10 +290,12 @@ pub fn build_ty(cx: &DocContext<'_>, did: DefId) -> Option<clean::Type> {
     }
 }
 
+// Passed `DefId` has to be a Trait item/path/reexport.
 pub fn build_impls(cx: &DocContext<'_>, did: DefId, attrs: Option<Attrs<'_>>) -> Vec<clean::Item> {
     let tcx = cx.tcx;
     let mut impls = Vec::new();
 
+    // for each implementation of a trait, build the clean::Item for that impl
```

Comment:
I'd rather fix https://github.com/rust-lang/rust/issues/76812 than add this comment, but in the meantime this is helpful.
```suggestion
    // for each inherent impl of a type (`impl T { ... }`), build the clean::Item for that impl
```